curp/src/server/cmd_worker/conflict_checked_mpmc.rs

```diff
@@ -59,7 +60,7 @@ pub(super) enum TaskType<C: Command> {
     /// Execute a cmd
     SpecExe(Arc<LogEntry<C>>, Option<C::Error>),
     /// After sync a cmd
-    AS(Arc<LogEntry<C>>, C::PR),
+    AS(Arc<LogEntry<C>>, Option<C::PR>),
     /// Reset the CE
```

Comment:
Why should we need to covert the `C::PR` into `Option<C::PR>`?

Reply:
Shutdown entry does not need C::PR